ci/spec.yml

```diff
@@ -134,19 +134,18 @@ jobs:
       - *copy_secret
 
       - name: Regenerate workflow YAML file and check for changes
-        run: |
-          set -e
-          pip install pyyaml
-          ./scripts/build_ensure_no_changes.sh ./ci/regen-workflow.py
+        run: ./scripts/build_ensure_no_changes.sh ./ci/regen-workflow.py
 
       # Build docs/sources
       - name: Build docs
         run: ./scripts/build_ensure_no_changes.sh ./scripts/build_docs.sh
       - name: Build sources
-        run: ./scripts/build_ensure_no_changes.sh ./scripts/build_sources.sh
+        run: |
+             source venv/bin/activate
+             ./scripts/build_ensure_no_changes.sh ./scripts/build_sources.sh
 
       # Check for changes to CI spec
-      - name: Check for chenges to CI spec
+      - name: Check for changes to CI spec
```

Comment:
This is already a part of #1148.

Reply:
Yup, I'll just rebase whichever one looses the race.